product_configurator_wizard/wizard/product_configurator.py

```diff
@@ -783,7 +786,8 @@ def action_config_done(self):
         if self.product_id:
             self.product_id.write({
                 'attribute_value_ids': [(6, 0, self.value_ids.ids)],
-                'value_custom_ids': [(6, 0, custom_vals)]
+                'value_custom_ids': map(lambda cv: (2, cv), self.product_id.value_custom_ids.ids) + \
+                                        self.product_id.product_tmpl_id.encode_custom_values(custom_vals),
```

Comment:
This is totally my bad, I didn't notice where the map ended and got confused (did this review after 10 hours of working on another project).

In the morning with coffee it looks much better :)

The only thing that I would suggest is maybe breaking the two write lists into variables so it's a bit more readable.

Other than that it makes perfect sense